share/doc/wake/logging.md

```diff
@@ -4,6 +4,13 @@ This article gives a high level overview of how wake logging works.
 Check out [print.wake](https://github.com/sifive/wake/blob/master/share/wake/lib/core/print.wake)
 for commented code.
 
+## Things that Might Appear on Output Streams
+
+* the stdout and stderr of executed Jobs (default to logInfo and logError, respectively, control with setPlanStdout, setPlanStderr)
+* The command line ('echo') of executed Jobs (default to logEcho, control with setPlanEcho)
+* The results of calling print/println (default to logInfo, control with printLevel, printlnLevel)
```

Comment:
I didn't actually know we had `print`/`printLevel`; you learn something new every day!  Apparently, though, the comments above the definitions are wrong since these don't actually go to `logInfo`:
```suggestion
* The results of calling print/println (default to logReport, control with printLevel, printlnLevel)
```

Reply:
excellent thanks